isaaclab_arena/affordances/placeable.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2025, The Isaac Lab Arena Project Developers (https://github.com/isaac-sim/IsaacLab-Arena/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import math
+import torch
+from typing import Literal
+
+import isaaclab.utils.math as math_utils
+from isaaclab.assets import RigidObject
+from isaaclab.envs.manager_based_env import ManagerBasedEnv
+from isaaclab.managers import SceneEntityCfg
+
+from isaaclab_arena.affordances.affordance_base import AffordanceBase
+
+
+class Placeable(AffordanceBase):
+    """Interface for placeable objects.
+    Placeable objects are objects that can be placed upright in a scene.
+    They are characterized by their upright axis and orientation threshold, which are used to determine if the object is placed upright.
+    - The upright axis is the axis in the object's local frame that is used to determine if the object is placed upright.
+    - The orientation threshold is the threshold for the angle between the upright axis and the world +Z direction.
+    """
+
+    def __init__(self, upright_axis_name: Literal["x", "y", "z"] = "z", orientation_threshold: float = 0.5, **kwargs):
```

Comment:
suggestion to add a unit to `orientation_threshold`. For example `orientation_threshold_deg` (if indeed the unit here is degrees.